ttnn/CMakeLists.txt

```diff
@@ -612,6 +614,9 @@ set(TTNN_PUBLIC_LINK_LIBRARIES
     metal_common_libs
     Metalium::Metal
     Boost::container
+    xtensor
+    xtensor-blas
+    xtl
```

Comment:
can you share a bit why do we need all 3 in ttnn?

Reply:
Discussed offline that it is ok to rely on ttnn for these imports, as is.